artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -707,6 +707,24 @@ public long getAddressMemoryUsage() {
       }
    }
 
+   @Override
+   public long getDiskStoreUsed() {
```

Comment:
I'm a bit confused between the metric and the math here... 

disk space - disk-usable? what that means!?

Reply:
If I understand correctly, disk usable represents the available space. Subtracting that from the total disk should give the disk used?

There is no documentation for disk usable method so I might be wrong.

Reply:
Ultimately the calls here are using [`java.nio.file.FileStore#getUsableSpace`](https://docs.oracle.com/javase/8/docs/api/java/nio/file/FileStore.html#getUsableSpace--) and [`java.nio.file.FileStore#getTotalSpace`](https://docs.oracle.com/javase/8/docs/api/java/nio/file/FileStore.html#getTotalSpace--) so the math looks right.

Reply:
Thanks for validating!